mod.go

```diff
@@ -16,11 +17,15 @@ import (
 
 // EnvLogLevel is the name of the environment variable to change the logging
 // level.
-const EnvLogLevel = "LLVL"
+const EnvLogLevel = "DBGSYNCLOG"
+const DebugFlag = "DBGSYNCON"
 
 const defaultLevel = zerolog.NoLevel
 
 func init() {
+	dbg := os.Getenv(DebugFlag)
+	DebugIsOn = dbg == "true"
```

Comment:
how about a case insensitive comparison?

Reply:
It's really just a matter of taste, no ? Maybe we should pick "1" or "0" as possibilities ;-)